mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/DebitCardFragment.java

```diff
@@ -95,7 +95,7 @@ public void verifyDebitCardSuccess(String otp) {
 
     @Override
     public void verifyDebitCardError(String message) {
-        showToast(message);
+        showToast(message); 
```

Comment:
```suggestion
        showToast(message);
```

Reply:
Hi! please remove this inessential 'space' after the semicolon. it is not required and removing it would decrease the number of files changed.